test/stateChangeCapture.test.ts

```diff
@@ -21,7 +21,7 @@ describe('stateChangeCapture', () => {
     eventStoreSpy.mockRestore();
   });
 
-  describe('sanitizePipelineName', () => {
+  describe('sanitizePipelineName', () => { // Take a regex Input and spit out a santiized string
```

Comment:
Yup, Will do.
JSDoc seems a good idea. Would you recommend doing it for a method
If it hasnt being used in the project?

Reply:
Using JSDoc syntax is a good starting point - happy to add it in incrementally as we see fit.

Whether we use the JSDoc to create separate doc pages is a different concern - but following the pattern is a good starting point.  There wasn't the requirement to generate JSDoc - since users at the moment would be working with the code itself (it isn't an API).